Mage/src/main/java/mage/abilities/costs/common/PayLoyaltyCost.java

```diff
@@ -33,8 +34,19 @@ public PayLoyaltyCost(PayLoyaltyCost cost) {
 
     @Override
     public boolean canPay(Ability ability, Ability source, UUID controllerId, Game game) {
+        int loyaltyCost = amount;
+        if (ability instanceof LoyaltyAbility) {
+            LoyaltyAbility copiedAbility = ((LoyaltyAbility) ability).copy();
+            game.getContinuousEffects().costModification(copiedAbility, game);
```

Comment:
Cost modification code must be called as two methods for compatibility (cause cost modification possible from the source/itself and from another effects):
```
sourceObject.adjustCosts(copyAbility, game);
game.getContinuousEffects().costModification(copiedAbility, game);
```